sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1200,6 +1200,11 @@ class Analyzer(override val catalogManager: CatalogManager)
                 val newRelation = multi.newInstance()
                 newRelation.copyTagsFrom(multi)
                 newRelation
+              case view @ View(_, _, child: LogicalPlan) =>
+                val newChild = child.mapExpressions {
+                  case alias: Alias => alias.newInstance()
+                }
+                view.withNewChildren(Seq(newChild))
```

Comment:
Although `MultiInstanceRelation` seems related to `Relation`, but `LogicalRDD`,`ExternalRDD` implements it.
So, could we let View implements `MultiInstanceRelation` too? @cloud-fan